packages/web/src/components/search/CategorySearch.js

```diff
@@ -359,17 +359,20 @@ class CategorySearch extends Component {
 
 	clearValue = () => {
 		this.setValue('', true);
+		this.onValueSelected('');
 	};
 
 	// only works if there's a change in downshift's value
 	handleOuterClick = () => {
 		this.setValue(this.state.currentValue, true);
+		this.onValueSelected();
```

Comment:
Why are we not passing any value here? Shouldn't we be passing the `currentValue` here as well?

Reply:
The function's parameter defaults to `this.state.currentValue`.

Reply:
cool beans 👍